cmd/edgectl/aes_install.go

```diff
@@ -437,166 +429,100 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
-	// Allow overriding the source domain (e.g., for smoke tests before release)
-	manifestsDomain := "www.getambassador.io"
-	domainOverrideVar := "AES_MANIFESTS_DOMAIN"
-	overrideMessage := "Downloading manifests from override domain %q instead of default %q because the environment variable %s is set."
-	if amd := os.Getenv(domainOverrideVar); amd != "" {
-		i.ShowWrapped(fmt.Sprintf(overrideMessage, amd, manifestsDomain, domainOverrideVar))
-		manifestsDomain = amd
+	namespace, err := i.kubeinfo.Namespace()
+	if err != nil {
+		i.Report("fail_no_cluster")
+		return fmt.Errorf(noCluster)
 	}
 
-	// Download AES manifests
-	crdManifests, err := getManifest(fmt.Sprintf("https://%s/yaml/aes-crds.yaml", manifestsDomain))
+	// Try to determine cluster type from node labels
+	ci := i.GetClusterInfo()
+	i.SetMetadatum("Cluster Info", "cluster_info", ci.name)
+
+	i.SetMetadatum("Cluster Info", "managed", "helm")
+
+	// create a new parsed checker for versions
+	chartVersion, err := helm.NewChartVersionRule("*")
 	if err != nil {
-		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-		return errors.Wrap(err, "download AES CRD manifests")
+		return err
```

Comment:
We should also report an event to Metriton, using `i.Report()` whenever we handle a failure scenario. 

I saw a few places where this is not the case when handling Helm errors here and bellow.

Reply:
Ok, I think I have added `i.Report()`s in all the new failures...